<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edituser</class>
 <widget class="QDialog" name="Edituser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Projects/Blink/icons/resources.qrc">
    <normaloff>:/png/marca-m.png</normaloff>:/png/marca-m.png</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>495</width>
     <height>151</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="Lblname">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtname">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>5</y>
      <width>420</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lblemail">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>45</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>e-mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtemail">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>40</y>
      <width>420</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lblcontractsince">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>80</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Admitido</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="DtEdtcontractsince">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>75</y>
      <width>170</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Lblpassword">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>115</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtpassword">
    <property name="geometry">
     <rect>
      <x>64</x>
      <y>110</y>
      <width>170</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Lblprofile">
    <property name="geometry">
     <rect>
      <x>261</x>
      <y>84</y>
      <width>30</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Perfil</string>
    </property>
   </widget>
   <widget class="SComboBox" name="CmbBxprofile">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>78</y>
      <width>27</width>
      <height>20</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
    <item>
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="Lblpassword_2">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>115</y>
      <width>59</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirmar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtConfirmPassword">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="topmenu">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>801</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/png/barra-superior-ok.jpg);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>541</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Cadastro de Usuários</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PshBtnCancel">
   <property name="geometry">
    <rect>
     <x>333</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnSave">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonResetarSenha">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Resetar Senha</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">scombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LnEdtname</tabstop>
  <tabstop>LnEdtemail</tabstop>
  <tabstop>DtEdtcontractsince</tabstop>
  <tabstop>LnEdtpassword</tabstop>
  <tabstop>LnEdtConfirmPassword</tabstop>
  <tabstop>CmbBxprofile</tabstop>
  <tabstop>PshBtnCancel</tabstop>
  <tabstop>PshBtnSave</tabstop>
 </tabstops>
 <resources>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
  <include location="../../Projects/Blink/icons/resources.qrc"/>
 </resources>
 <connections/>
</ui>
